### ISSUE TITLE
Broken: Adding fields/-sets to Space Dexterity type via GUI

### ISSUE THREAD

**Issue by petri** [collective/collective.spaces#2]
Trying to add fields or fieldsets to a Space, using the Dexterity control panel GUI on latest Plone 4.3.2 & latest Dexterity results (upon new field/-set submission) in:

```
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module plone.z3cform.layout, line 66, in __call__
  Module plone.z3cform.layout, line 50, in update
  Module z3c.form.form, line 210, in update
  Module plone.app.z3cform.csrf, line 21, in execute
  Module z3c.form.action, line 98, in execute
  Module z3c.form.button, line 315, in __call__
  Module z3c.form.button, line 170, in __call__
  Module z3c.form.form, line 239, in handleAdd
  Module z3c.form.form, line 247, in createAndAdd
  Module plone.schemaeditor.browser.schema.add_field, line 32, in add
  Module plone.schemaeditor.utils, line 28, in non_fieldset_fields
  Module zope.interface.interface, line 76, in getTaggedValue
KeyError: 'plone.supermodel.fieldsets'
```

The same thing happens to a content type created using "clone" in the UI.

**Comment by petri** [comment]
Adding behaviours works ok.

**Comment by davidjb** [comment]
Thanks for the bug report -- I've reproduced the issue and it stems from plone.schemaeditor.  The issue was fixed in version 1.3.5, so if you add a version pin like:

```
[versions]
...
plone.schemaeditor = 1.3.5
```

into your `buildout.cfg`, then this will resolve the errors.  However, there's an issue at plone/plone.app.dexterity#33 whereby any through-the-web changes are lost after you restart your Plone instance. Feel free to comment and follow up on that issue, much like I will shortly.
